src/el/input_parser.star

```diff
@@ -19,7 +19,13 @@ _DEFAULT_ARGS = {
     "max_mem": 0,
 }
 
-_DEFAULT_BUILDER_ARGS = _DEFAULT_ARGS | {"key": None}
+_DEFAULT_FLASHBLOCKS_WS_PORT = 1111
+
+_DEFAULT_BUILDER_ARGS = _DEFAULT_ARGS | {
+    "key": None,
+    # Flashblocks-related defaults for builders
+    "flashblocks_miliseconds_per_slot": 250,
```

Comment:
I think it's a _milli_second in english (i think just because otherwise it would be pronounced like miley cyrus)